pkgs/development/libraries/libverto/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, glib
+, libev
+, libevent
+, pkg-config
+, glibSupport ? true
+, libevSupport ? true
+, libeventSupport ? true
+}:
+
+let
+  inherit (lib) optional;
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libverto";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "latchset";
+    repo = "libverto";
+    rev = finalAttrs.version;
+    hash = "sha256-csoJ0WdKyrza8kBSMKoaItKvcbijI6Wl8nWCbywPScQ=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkg-config
+  ];
+
+  buildInputs =
+    optional glibSupport glib
```

Comment:
```suggestion
  buildInputs = optional glibSupport glib
```

Reply:
It makes indentation a bit strange in Emacs `nix-mode`: it tries to align all below the previous string, making something like 

```nix
buildInputs = optional glibSupport glib
                         ++ optional libevSupport libev
                         ++ optional libeventSupport libevent;
```

Reply:
> It makes indentation a bit strange in Emacs `nix-mode`

Now I know which program does those strange indentations. We always intend with 2 spaces.